src/codecs/html/README.md

```diff
@@ -38,3 +38,218 @@ Some of the main external dependencies:
+        * references: `ol itemprop='stencila:CiteGroup'` of `li ul` Cite itemtypes (see above) with `ol` authors, `a href` target, `span` title
+        * content: `article`/`div` with `sections`
+        * dateCreated: `span time`
+        * dateModified: `span time`
+        * datePublished/date: `span time`
+        * editors: `ol` of `span` Person `name` (or combined `givenNames`+`familyNames`)
+        * funders: `ol` of `span` Person `name` (or combined `givenNames`+`familyNames`)/Organization `legalName`.
+        * isPartOf: `data`?
+        * licenses: `ol` of `a href`
+        * parts/hasPart: `ol` of figure/table/object/URI (maybe unnecessary for styling)
+        * publisher: `span` Person `name` (or combined `givenNames`+`familyNames`)/Organization `legalName`.
+        * text: `data`? (maybe unnecessary for styling)
+        * title/headline: `h1`
+        * version: `span`
+* DataTable:
+    * `table` or `figure table` `itemtype='stencila:Datatable'`
+    * columns: probably want to convert these DatatableColumns into rows when encoding into HTML, so they're made up of `tr` and `td`/`th`. Maybe keep as `data`?
+        * values: `td`
+        * schema (items, uniqueItems): `data`? (maybe unnecessary for styling)
+        * name: `th`?
+* Organization:
+    * `div` `itemtype='schema:Organization'`
+    * legalName: set as `div` textContent
+    * Need to determine which itemprops are important to show. Maybe just `address`, `brand`, `departments`, and `legalName`.
+    * itemprops:
+        * address: `address` `itemprop='schema:address'`
+        * brands: `ul` of `picture`/`img` (see `Brand` above)
+        * contactPoints: `ul` of `li address` (see `ContactPoint` above)
+        * departments: `ul` of `span` Organization `legalName`
+        * funders: `data`? (maybe unnecessary for styling, since `funders` included seperately in CreativeWork)
+        * parentOrganization: `data`? (maybe unnecessary for styling)
+* Periodical:
+    * dateStart, dateEnd: `data`? (maybe unnecessary for styling)
+    * issn: `p`
+* Person:
+    * `div` `itemtype='schema:Person'`
+    * Need to determine which itemprops are important to show. Maybe just `affiliations`, `familyNames`, `givenNames`, `name`, `honorificPrefix`, `honorificSuffix`, and `jobTitle`.
+    * itemprops:
+        * address: `address` `itemprop='schema:address'` or `data`? (maybe unnecessary for styling)
+        * affiliations: `ul` of `span` Organization `legalName` + `departments`
+        * emails: `data`? (maybe unnecessary for styling)
+        * familyNames: `ul` of `span`
+        * funders: `data`? (maybe unnecessary for styling)
+        * givenNames: `ul` of `span`
+        * honorificPrefix: `span`
+        * honorificSuffix: `span`
+        * jobTitle: `span`
+        * memberOf: `ul` of `span` Organization `legalName` + `departments`, or `data`? (maybe unnecessary for styling)
+        * telephoneNumbers: `data`? (maybe unnecessary for styling)
```

Comment:
`ul` of `a` like this `<a href="tel:8675309">`?

Reply:
Do you think we need to display phone numbers when including a `Person` in a reproducible document/article?

It looked like the main use case for `Person` is to display lists of author names, publishers, editors, etc. For `Organization`, I think it makes sense to display phone numbers. I'm not sure if authors/individuals would want their phone numbers publicly available.

Reply:
I have no opinion one way or the other, just pointing out the use of tel: attribute.